scripts/install-deps.sh

```diff
@@ -156,6 +161,25 @@ do_install_githooks() {
     fi
 }
 
+do_install_openssl() {
+    local openssl_dir=$work_folder/openssl-1.1.1f
```

Comment:
why 1.1.1f?  There were many CVE vulnerabilities fixed since `f` as can be seen here: https://www.openssl.org/news/vulnerabilities-1.1.1.html

I think it should be latest to get all the security fixes, which is `1.1.1u`
https://www.openssl.org/source/openssl-1.1.1u.tar.gz